library/wizard/terminate_reason.py

```diff
@@ -0,0 +1,52 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class TerminateReasonLibrary(models.TransientModel):
+    _inherit = "terminate.reason"
+
+    info = fields.Text("Message", help="Enter message")
+
+    @api.model
+    def default_get(self, fields):
+        """Override method to display message if student has issued book
+        while terminate student"""
+        res = super(TerminateReasonLibrary, self).default_get(fields)
+        student_obj = self.env["student.student"].browse(
```

Comment:
student_obj i didn't find any more use of this variable.
so, you can directly using  self._context.get("active_id", False)
instead of browse the that